### PR TITLE
Update mining faq

### PR DESCRIPTION
**Pull request by kimihiro64**
Adds a FAQ section and guidance on submitting bug reports

### CODE REVIEW

**Comment by lyoshenka** [comment]
@kimihiro64 thanks for the faq writeup. can you please break up the faq into separate files? each file in /posts/faq is its own page with its own question and answer. look at the files in https://github.com/lbryio/lbry.io/tree/master/posts/faq 

We can combine multiple questions into a single page, but I'd rather do that programmatically (maybe via  a category or tag in the frontmatter). 

Thanks for your help :clap:

**Comment by kimihiro64** [comment]
Oh, I see. I didn't even know we had that centralized FAQ page. Mind if I put a link to it on home page? I will break up the questions.

**Comment by lyoshenka** [comment]
We haven't linked to it yet because we're still figuring out how it should be organized. We'll link to it soon though. Where do you think the link should go?

**Comment by kimihiro64** [comment]
I was thinking right next to the Learn link.

**Comment by kimihiro64** [comment]
Made the necessary changes to break out into separate .md files, removed duplicate info and added some info elsewhere in the FAQ where it made sense

File: posts/faq/custom-ui.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ title: How do I specify a web-UI to use?
 
 If the files for the UI you'd like to use are stored locally on your computer, start lbry with the `--ui` flag:
 
-    lbrynet-daemon --ui=/full/path/to/ui/files/root/folder
+    `lbrynet-daemon --ui=/full/path/to/ui/files/root/folder`
```
Comment (lyoshenka) [inline posts/faq/custom-ui.md]: @kimihiro64 when text is indented via 4 (or more) spaces, its in a code block. there's no need to wrap that text in ticks. in fact, those ticks will show up as characters in the code.

File: posts/faq/how-to-backup-wallet.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+title: How do I mine backup my wallet?
+---
+
+You can use `lbrycrd-cli backupwallet <pathtobackup>` to back up your wallet, or alternatively you can copy the wallet.dat file manually to anywhere you choose [important directories](https://lbry.io/faq/lbry-directories).
```
Comment (lyoshenka) [inline posts/faq/how-to-backup-wallet.md]: i like the links to "important directories" everywhere, but you should either link to it as part of the sentence, or put it on a separate line. For example:

> You can use `lbrycrd-cli backupwallet <pathtobackup>` to back up your wallet, or alternatively you can copy the wallet.dat file manually to anywhere you choose.

> See also: [important directories](https://lbry.io/faq/lbry-directories)

**Comment by kimihiro64** [comment]
Ah, nice to know, still a markdown nub

File: posts/faq/how-to-report-bugs.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+---
+title: How do I report bugs?
+---
+
+Have a technical problem? If you're having an issue with install of the UI/miner, or something involving an invite code, please join the slack at https://lbry.slack.com and in channel #bugs post a SINGLE detailed note with what exactly is going on.  If you can get to the LBRY UI after installing, but can't watch a video or have another problem, please click the three horizontal lines in the top right hand corner of the LBRY UI, click Help, then click the Send Us a Bug Report link.
```
Comment (lyoshenka) [inline posts/faq/how-to-report-bugs.md]: Please include this too:

"If you need an invite to the LBRY slack, you can get one at https://slack.lbry.io".

**Comment by lyoshenka** [comment]
Thanks. I made a few small comments inline, but its almost ready to go.

File: posts/faq/api-help.md [hunk 4]
```diff
@@ -2,7 +2,15 @@
 title: How do I see the list of API functions I can call, and how do I call them?
 ---
 
-Here is an example script to get the documentation for the various API calls. To use any of the functions displayed, just provide any specified arguments in a dictionary.
+Here is an example script to get the documentation for the various API calls. To use any of the functions displayed, just provide any specified arguments in a dictionary. Many (though not all) of the calls are the same as those for bitcoin core, which are documented [here](https://en.bitcoin.it/wiki/Original_Bitcoin_client/API_calls_list).
+
+If for some reason you can't get lbrycrd-cli working to make these calls, make sure you passed -server when starting lbrycrd. If it still doesn't work, you can use this command if running lbrycrd with -server:
+
+`curl --user USER:PASSWORD --data-binary '{"jsonrpc": "1.0", "id":"curltest", "method": "COMMAND", "params": [] }' -H 'content-type: text/plain;' http://127.0.0.1:9245/`
```
Comment (lyoshenka) [inline posts/faq/api-help.md]: Last nit: For long commands, its better to put them in a code block (using 4 indents) than to use the tick marks. Tick marks are best for short bits of code that are inline.

**Pull request merged**